src/checkstyle/checks.xml

```diff
@@ -171,6 +171,11 @@
                 STAR, STAR_ASSIGN, TYPE_EXTENSION_AND" />
         </module>
 
+        <module name="NoWhitespaceAfter">
```

Comment:
After the fix in this PR, this rule passes with the defaults, which includes several other tokens such as `++`. Also, `allowLineBreaks` defaults to false. So we can make this:

```xml
<module name="NoWhitespaceAfter" />
```

And put this with the `NoWhitespaceBefore` rule.